ladybug/legend.py

```diff
@@ -0,0 +1,877 @@
+        return self._continuous_colors
+
+    @continuous_colors.setter
+    def continuous_colors(self, cont_cols):
+        if cont_cols is not None:
+            assert isinstance(cont_cols, bool), \
+                'Expected boolean for continuous_colors. Got {}.'.format(type(cont_cols))
+            self._continuous_colors = cont_cols
+        else:
+            self._continuous_colors = True
+
+    @property
+    def continuous_legend(self):
+        """Boolean noting whether legend is drawn as a gradient or discrete segments."""
+        return self._continuous_legend
+
+    @continuous_legend.setter
+    def continuous_legend(self, cont_leg):
+        if cont_leg is not None:
+            assert isinstance(cont_leg, bool), \
+                'Expected boolean for continuous_legend. Got {}.'.format(type(cont_leg))
+            self._continuous_legend = cont_leg
+        else:
+            self._continuous_legend = False
+
+    @property
+    def title(self):
+        """Get or set the text for the title of the legend."""
+        return self._title
+
+    @title.setter
+    def title(self, title):
+        if title is not None:
+            assert isinstance(title, str), \
+                'Expected string for title. Got {}.'.format(type(title))
+            self._title = title
+            self._is_title_default = False
+        else:
+            self._title = ''
+            self._is_title_default = True
+
+    @property
+    def ordinal_dictionary(self):
+        """Get or set a dictionary that maps values to text categories."""
+        return self._ordinal_dictionary
+
+    @ordinal_dictionary.setter
+    def ordinal_dictionary(self, o_dict):
+        if o_dict is not None:
+            assert isinstance(o_dict, dict), \
+                'Expected dictionary for ordinal_dictionary. Got {}.'.format(
+                    type(o_dict))
+            for key in o_dict.keys():
+                assert isinstance(key, int), \
+                    'Expected integer for ordinal_dictionary key. Got {}.'.format(
+                        type(key))
+        self._ordinal_dictionary = o_dict
+
+    @property
+    def number_decimal_places(self):
+        """Get or set the number of decimal places in the legend text."""
+        return self._number_decimal_places
+
+    @number_decimal_places.setter
+    def number_decimal_places(self, n_dec):
+        if n_dec is not None:
+            assert isinstance(n_dec, int), \
+                'Expected integer for number_decimal_places. Got {}.'.format(type(n_dec))
+            self._number_decimal_places = n_dec
+        else:
+            self._number_decimal_places = 2
+
+    @property
+    def include_larger_smaller(self):
+        """Boolean noting whether > and < should be included in legend segment text."""
+        return self._include_larger_smaller
+
+    @include_larger_smaller.setter
+    def include_larger_smaller(self, lgsm):
+        if lgsm is not None:
```

Comment:
I saw this a couple of times and didn't leave a comment but just as suggestion I would say just a quick check is enough for a Boolean.

```
self._include_larger_smaller = bool(lgsm)
```

Reply:
Noted and implemented.